docs/experiments/extract-mode.md

```diff
@@ -33,7 +33,7 @@ rules:
       - pattern: eval ...
 ```
 
-Usually, this rule would only be run against Bash files. However, our project might contain Dockerfiles or Python scripts which invoke Bash commands&mdash;without an extract mode rule Semgrep won't know about any contained Bash, and therefore, will **not** run any Bash rules against those contained commands.
+Usually, Semgrep uses this rule only against Bash files. However, a project might contain Dockerfiles or Python scripts that invoke Bash commands&mdash;without an extract mode rule, Semgrep does **not** run any Bash rules against commands contained in files of different languages.
 
 However, with extract mode we can provide Semgrep with instructions on how to extract any Bash commands we might be using in a Docker `RUN` instruction or as an argument to Python's `os.system` standard library function.
```

Comment:
@kopecs just suggesting a little simplification again, to avoid the use of `we`.

```suggestion
However, with extract mode, Semgrep can obtain instructions on how to extract any Bash commands used in a Docker `RUN` instruction or as an argument to Python's `os.system` standard library function.
```

Reply:
My concern with the proposed phrasing would be that it implies that Semgrep would somehow automatically obtain these instructions, instead of them being provided explicitly by the user or in a rulepack. What about:

"However, with extract mode, you can provide Semgrep with instructions on how to extract any Bash commands used in a Docker `RUN` instruction or as an argument to Python's `os.system` standard library function."

Reply:
I agree!

```suggestion
However, with extract mode, you can provide Semgrep with instructions on how to extract any Bash commands used in a Docker `RUN` instruction or as an argument to Python's `os.system` standard library function.
```